<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendFileDialog</class>
 <widget class="QDialog" name="SendFileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>135</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SendFileDialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="text">
    <string>选择发送文件</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="openBtn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendBtn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeBtn">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>371</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>openBtn</sender>
   <signal>clicked()</signal>
   <receiver>SendFileDialog</receiver>
   <slot>on_Open()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>22</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sendBtn</sender>
   <signal>clicked()</signal>
   <receiver>SendFileDialog</receiver>
   <slot>on_Send()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeBtn</sender>
   <signal>clicked()</signal>
   <receiver>SendFileDialog</receiver>
   <slot>on_Close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>300</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>359</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_Open()</slot>
  <slot>on_Send()</slot>
  <slot>on_Close()</slot>
 </slots>
</ui>
